src/dom-to-image.js

```diff
@@ -244,6 +259,86 @@
                         });
                     }
                 }
+
+                if(domtoimage.impl.options.scrollFix &&
+                    (original.scrollTop || original.scrollLeft)) {
+                    // Setup container for absolute positioning of children
+                    clone.style.position = 'relative';
+                    clone.style.overflow = 'hidden';
+                    clone.style.width = original.offsetWidth + 'px';
+                    clone.style.height = original.offsetHeight + 'px';
+                    var scrollTopRemaining = original.scrollTop > 0 ? original.scrollTop : null;
+                    var scrollLeftRemaining = original.scrollLeft > 0 ? original.scrollLeft : null;
+                    var originalIsPositionRelative = originalStyle['position'] === 'relative';
+                    var computedStylesCache = {};
+                    var boundingRectCache = {};
+                    
+                    // Loop through children and set position based on original
+                    // childs position and original containers scroll position
+                    for(var i = 0; i < clone.children.length; i++) {
+                        // Make sure this element is stylable
+                        if(typeof(clone.children[i]) === 'undefined' ||
```

Comment:
We should do a `var cloneChild = clone.children[i];` to avoid all the multiple times we compute it in following code.